opendevin/runtime/docker/ssh_box.py

```diff
@@ -748,7 +744,8 @@ def close(self):
             try:
                 if container.name.startswith(self.container_name):
                     if config.persist_sandbox:
-                        container.stop()
+                        if not config.sandbox.fast_boot:
+                            container.stop()
```

Comment:
I still don't understand this, sorry. Does this mean that `fast_boot` is the same with `not initialize_plugins`?

Reply:
would this work with non-persistent sandboxes (the default)?
wouldn't a new session create a new container anyways and leave the old "abandonded" here?
@enyst

Reply:
I guess it wouldn't work well, but the lines above limit it to `persist_sandbox`.

Reply:
for the first time, if we set initialize_plugins, not initialize_plugins will fail and sandbox stopped -> jupyter server also stopped. (then we need to initialize_plugins again) -> loop

Reply:
What happens if `initialize_plugins` and `fast_boot` are both true, or are both false?

Reply:
The purpose of `fast_boot` is to not stop the container for a persistent sandbox.

Reply:
if initialize_plugins and fast_boot are set to true; it will initialize_plugins and don't stop the container.
if both are set to false, if the plugin is already initialized, and if the sandbox was not stopped before, it will work. But for the next run, the jupyter server will be stopped as the sandbox was stopped. So, it won't work.